client/src/templates/Challenges/components/tool-panel.css

```diff
@@ -14,8 +14,35 @@
   padding: 0 2px;
 }
 
-.tool-panel-group-mobile .btn-block + .btn-block {
+.tool-panel-group-mobile > .btn-block {
   margin: 0 2px 0 0;
   padding-left: 8px;
   padding-right: 8px;
 }
+
+.tool-panel-group .btn-group {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+.tool-panel-group-mobile .btn-group {
+  display: block;
+  width: 100%;
+  margin: 0 2px 0 2px;
+}
+
+.tool-panel-group-mobile .btn {
+  margin-bottom: 0 !important;
+}
```

Comment:
do we need this, it seems like line 18 is taking care of this?

Reply:
other than that it LGTM

Reply:
I think there was some styling in global.css taking precedence that caused the button spacing to be off.

Reply:
I tried it locally and gitpod. removing it did not change the style. could you reconfirm and remove if not necessary? thank you

Reply:
This is the issue I was seeing. When removed there is a 10px margin below the buttons, leading to page overflow and the buttons can be scrolled past.

![Screenshot from 2019-11-08 12-57-31](https://user-images.githubusercontent.com/24216368/68506813-ae54e200-0227-11ea-9191-f300ee1752d5.png)

Actually even without the bottom margin, there is still a 1-2 px overflow, not really noticeable if on a mobile device, but can lead to a double vertical scrollbar on desktop. I can't figure out what's causing it though.

Reply:
hmm, for some reason i could not reproduce either.